CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ project(ufs-weather-model
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 
-set(32BIT           OFF CACHE BOOL "Enable 32BIT (single precision arithmetic in dycore)")
+set(32BIT           ON  CACHE BOOL "Enable 32BIT (single precision arithmetic in dycore)")
```

Comment:
@DomHeinzeller quick noob question....does the "CACHE" here mean that we could override this setting at a higher CMake level? Like if I specify this line (20) in the ufs-srweather-app/CMakeList.txt, will it take that setting when building the model? Would I maybe exclude "CACHE" when doing that to override?

Reply:
I honestly don't know, but I think you are correct.